compiler/circle-mpqsolver/CMakeLists.txt

```diff
@@ -54,3 +57,4 @@ target_link_libraries(circle_mpqsolver_test luci_testhelper)
 target_link_libraries(circle_mpqsolver_test luci_export)
 target_link_libraries(circle_mpqsolver_test luci_interpreter)
 target_link_libraries(circle_mpqsolver_test dio_hdf5)
+target_link_libraries(circle_mpqsolver_test luci_import)
```

Comment:
Can you move this line above `luci_export` ?

Reply:
@seanshpark 
done